torch/onnx/utils.py

```diff
@@ -1314,6 +1314,70 @@ def _validate_dynamic_axes(dynamic_axes, model, input_names, output_names):
             dynamic_axes[key] = value_dict
 
 
+# This function export torch::jit::Graph object
+# to serialized ONNX ModelProto.
+# In addition to the Python tests of this function
```

Comment:
I wouldn't mention the tests here or how to find them. That's pretty standard for any function and doesn't add useful context.